src/app/services/annotator/annotator.service.ts

```diff
@@ -0,0 +1,15 @@
+import { Injectable } from '@angular/core';
+import { Subject } from 'rxjs';
+
+@Injectable({
+  providedIn: 'root',
+})
+export class AnnotatorService {
+  textSelection: Subject<object> = new Subject();
+
+  getTextSelection() {
```

Comment:
the name suggests you return something, but you do not.
Either change the name or return te text selection